cmake/modules/FindStdFilesystem_test.cc

```diff
@@ -1,13 +1,11 @@
 #if __has_include(<filesystem>)
 #include <filesystem>
-namespace fs = std::filesystem;
-#elif __has_include(<experimental/filesystem>)
-#include <experimental/filesystem>
-namespace fs = std::experimental::filesystem;
 #else
 #error std::filesystem not available!
 #endif
```

Comment:
Nit: why not just unconditionally include `<filesystem>` here now?

Reply:
good point. because i wanted to have the "std::filesystem not available!" message, if somebody complains that Ceph does not build more and paste `CMakeFiles/CMakeError.log`, i could just search for it.

Reply:
after a second thought, hmm, probably the chance of such a failure is rare. as we already check the compiler version.

Reply:
> unconditionally include <filesystem> here now?

updated and repushed.